Sources/Internal/ScrollMetricsMutator.swift

```diff
@@ -108,26 +112,15 @@ final class ScrollMetricsMutator {
 
   // MARK: Private
 
-  /// A content size multiplier of 15 is large enough to prevent a high-velocity scroll from causing us to bump into an edge prematurely.
-  private static let contentSizeMultiplier: CGFloat = 15
-  private static let loopingRegionSizeDeterminingMultiplier: CGFloat = 1 / 3
+  private static let contentSize: CGFloat = 30_000 // 10,000 padding from min and max offset
+  private static let minimumContentOffset: CGFloat = 10_000 // 1/3 of content size
+  private static let maximumContentOffset: CGFloat = 20_000 // 2/3 of content size
+  private static let loopingRegionSize: CGFloat = 10_000 // Distance between min and max offset
```

Comment:
There was no good reason for these looping key-offsets to be dynamic based on the current bounds before - it just made things way more complicated to understand and debug, since we'd have to invalidate the whole `scrollMetricsMutator` anytime our bounds changed.